hazelcast/src/main/java/com/hazelcast/client/impl/ClientEndpointImpl.java

```diff
@@ -307,6 +308,7 @@ private MetricDescriptor enhanceDescriptor(MetricDescriptor descriptor, long tim
                             .withIncludedTarget(MANAGEMENT_CENTER)
                             // we add "client" and "timestamp" tags for MC
```

Comment:
nit: this comment does not mention client name, so it's better to change it to smth like `we add some tags for MC`